packages/animations/lib/src/open_container.dart

```diff
@@ -593,7 +597,9 @@ class _OpenContainerRoute extends ModalRoute<void> {
     bool delayForSourceRoute = false,
   }) {
     final RenderBox navigator =
-        Navigator.of(navigatorContext).context.findRenderObject();
+        Navigator.of(navigatorContext, rootNavigator: useRootNavigator)
```

Comment:
nit: use a trailing comma so that the formatter doesn't reformat the entire block
```suggestion
        Navigator.of(navigatorContext, rootNavigator: useRootNavigator,)
```